src/property.cpp

```diff
@@ -62,7 +50,7 @@ std::istream& operator>>(std::istream& is, Properties& props)
         }
         // Extract the key.
         auto end = line.find('=', begin);
-        std::string key = line.substr(begin, end - begin);
+        auto key = line.substr(begin, end - begin);
```

Comment:
I think you can keep `std::string` here.
The line above use `auto` because the iterator type are very verbose, but for `std::string` I think it's clearer to keep it.

Reply:
it's done